lua/wire/tool_loader.lua

```diff
@@ -426,10 +426,10 @@ function WireToolSetup.setToolMenuIcon( icon )
 end
 
 -- makes a new TOOL
---  s_mode: Tool_mode, same as the old tool lua file name, minus the "wire_" part
---  s_name: Proper name for the tool
---  s_class: For tools that make a device. Should begin with "gmod_wire_". Can be nil if not using WireToolObj.LeftClick or WireToolSetup.BaseLang
---  f_toolmakeent: Server side function for making the tools device. Can be nil if not using WireToolObj.LeftClick
+-- s_mode: Tool_mode, same as the old tool lua file name, minus the "wire_" part
+-- s_name: Proper name for the tool
+-- s_class: For tools that make a device. Should begin with "gmod_wire_". Can be nil if not using WireToolObj.LeftClick or WireToolSetup.BaseLang
+-- f_toolmakeent: Server side function for making the tools device. Can be nil if not using WireToolObj.LeftClick
```

Comment:
If you see the previous occurrence of this, you will see that the in-comment indentation is a bit inconsistent. This one only has space of indentation whereas the previous one had two.